src/kv/test/kv_bench.cpp

```diff
@@ -62,7 +62,7 @@ static void deserialise(picobench::state& s)
   Store kv_store2;
 
   auto secrets = create_ledger_secrets();
-  auto encryptor = std::make_shared<ccf::TxEncryptor>(0x1, secrets);
+  auto encryptor = std::make_shared<ccf::RaftTxEncryptor>(0x1, secrets);
```

Comment:
Why 0x1 here and 1 in the next file?

Reply:
I don't know it was like this when I found it. If no one can remember why I will change it to 1 here too.

Reply:
It's the same value, it's just... odd? :)